bdai_ros2_wrappers/test/test_logging.py

```diff
@@ -1,15 +1,80 @@
 # Copyright (c) 2023 Boston Dynamics AI Institute Inc.  All rights reserved.
 import logging
-from typing import Optional
+import threading
+from typing import List, Optional
```

Comment:
we could use `list` and not import this.

Reply:
Same as above about 3.8